src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private final Object[] keys = new Object[MAX_STORAGE_SIZE];
+    private final Object[] values = new Object[MAX_STORAGE_SIZE];
+    private int size;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if ((key == keys[i]) || (keys[i] != null && keys[i].equals(key))) {
+                values[i] = value;
+                return;
+            }
+        }
+        keys[size] = key;
+        values[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < size; i++) {
+            if (key == null) {
+                if (key == keys[i]) {
+                    return (V) values[i];
+                }
+                continue;
+            }
+            if (keys[i] != null && keys[i].equals(key)) {
+                return (V) values[i];
+            }
```

Comment:
```suggestion
            if (key == keys[i] || (keys[i] != null && keys[i].equals(key))) {
                return values[i];
            }
```
You can easily simplify if's

Reply:
Really) looks much more simple
Thanks